### DISCUSSION TITLE
ERR_CONNECTION_REFUSED : crbug/1173575, non-JS module files deprecated.

### DISCUSSION THREAD

**Discussion by Yakito** [gatsbyjs/gatsby#34876] [Help]
Hi! I've been struggling with trying to solve this for the past few days. Maybe someone can help.

I got a new server on Vultr and I am simply trying to install Gatsby. Everything goes well during installation, but when I try to access my site on `[server-ip]:8000 ` I get `ERR_CONNECTION_REFUSED`

When I check the inspector I am getting 1 warning

`crbug/1173575, non-JS module files deprecated.` 

There are some reports on crbug/1173575 around the web but no clear solutions. Also, I believe this must not be something common because otherwise everyone would be talking about it.

Any ideas or suggestion on what should I do to make Gatsby work?

Thanks in advance!

EDIT: BTW, I am sure the port is open.
EDIT 2: Just tried with a Lightsail instance, and the same result. I wonder why there aren't more people complaining about the same. What I am doing wrong

**Comment by Yakito** [comment]
no one else having this issue?

**Reply by Arkady-Skvortsov** [reply]
Me, am I up full stack Vue/Express in Docker; And I dunnow why I catch it

**Comment by rodrickcalvin** [comment]
I am getting a similar issue. When I stop the local server `http://localhost:8000/` and re-run it. It some times works.
It at times fails when I make a change and it tries to reload
